doc/classes/AnimationNodeStateMachinePlayback.xml

```diff
@@ -93,4 +93,18 @@
 	<members>
 		<member name="resource_local_to_scene" type="bool" setter="set_local_to_scene" getter="is_local_to_scene" overrides="Resource" default="true" />
 	</members>
+	<signals>
+		<signal name="node_finished">
+			<param index="0" name="node" type="StringName" />
+			<description>
+				Emitted when the [param node] finishes playback. If [param node] is a state machine set to grouped mode, its signals are passed through with its name prefixed.
+			</description>
+		</signal>
+		<signal name="node_started">
```

Comment:
```suggestion
		<signal name="state_started">
```
I made the same comment in https://github.com/godotengine/godot/pull/102165#pullrequestreview-2607672617. Avoid using the word "node".